lib/common/db-renderable-content.js

```diff
@@ -118,6 +120,11 @@ function dbRenderableContentServiceFactory(
 
     DbRenderableContentService.prototype.render = function render (name, options, scope) {
         return this.get(name, scope).then(function (template) {
+            if (!template) {
```

Comment:
@benbp I take it back.  `template` is an object; the renderable string is referenced by the `contents` property, so an empty string will still pass this test.